libclambcc/Common/ClamBCRegAlloc.cpp

```diff
@@ -141,8 +146,19 @@ bool ClamBCRegAlloc::runOnFunction(Function &F)
                         ClamBCStop("Cast from pointer to non-pointer element",
                                    BCI);
                     }
+#if 0
                     SrcTy = SPTy->getElementType();
                     DstTy = DPTy->getElementType();
+#else
+
+                    assert (0 && "removed deprecated getPointerElementType calls");
```

Comment:
A little confused about this assert message.  I guess it's a reminder?

Reply:
yeah, just a reminder.  SPTy->getElementType has been removed, so I need to go back and figure out what those instructions are.  The easiest way to do that is dumping the instructions when I get to that point, so I put the assert in to remember when I get to testing.